template/ios/Podfile

```diff
@@ -15,7 +15,7 @@ target 'RNBaseProject' do
     # Hermes is now enabled by default. Disable by setting this flag to false.
     # Upcoming versions of React Native may rely on get_default_flags(), but
     # we make it explicit here to aid in the React Native upgrade process.
-    :hermes_enabled => true,
+    :hermes_enabled => false,
```

Comment:
Let's enable Hermes because of its advantage. Also, we have implemented mmkv so we cannot debug on Chrome anymore.